### ISSUE TITLE
It's not moving on my phone.

### ISSUE THREAD

**Issue by SaipanDamashii** [huacnlee/PokemonGoMove#46]
I ran the project and then selected Simulate Location to PokemonLocation, but when I move the map around the player character on my phone isn't moving with it. Help please, I'm not sure what info you need from me but I'll get it to you once I am aware of what I'm doing wrong, etc.

**Comment by diddy81** [comment]
did you miss this part ?
For the very first time of running the project, you will need to allow add terminal to be accessible. System Preferences > Security & Privacy > Privacy > Accessibility > Terminal (checked)

also be sure you have Debug->Simulate Location->PokemonLocation is checked

**Comment by MeGaPk** [comment]
@SaipanDamashii 
Please check it
https://github.com/huacnlee/PokemonGoMove/issues/37

**Comment by SaipanDamashii** [comment]
No, I made sure to follow the accessibility part, as well as the PokemonLocation simulation.  It wasn't working earlier, so I exited out of everything...and now when I try to re-do the whole process, it's telling me that 

>  The directory '/Users/kelvin/Library/Caches/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
> The directory '/Users/kelvin/Library/Caches/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
> Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'

The other part works fine, with the localhost:3001 part, but the bottom part, the Pokemap, is grayed out/not working, and I'm guessing it's because I can't seem to get the above part working anymore....what am I doing wrong?

**Comment by SaipanDamashii** [comment]
Also, I should probably mention that if I try to skip the requirements part, it then returns this error:

> python: can't open file 'example.py': [Errno 2] No such file or directory
